### PR TITLE
Update Helm chart to support deployment/pod labels

### PR DESCRIPTION
**Pull request by romanwozniak**
### Context: 

This MR adds support for user-provided labels to be applied to Merlin deployment/pods.

### CODE REVIEW

File: charts/merlin/templates/merlin-configmap.yaml [hunk 1]
```diff
@@ -6,16 +6,10 @@ metadata:
...
-    heritage: {{ .Release.Service }}
     helm.sh/chart: {{ include "merlin.chart" . }}
     app.kubernetes.io/name: {{ include "merlin.name" . }}
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
-    gojek.com/app: merlin
-    gojek.com/team: dsp
-    gojek.com/stream: dsp
-    gojek.com/orchestrator: gitlab
-    gojek.com/environment: {{ .Values.merlin.environment }}
```
Comment (karzuo) [inline charts/merlin/templates/merlin-configmap.yaml:18]: where are these shifted to?

Comment (karzuo) [reply charts/merlin/templates/merlin-configmap.yaml:18]: ok nvm. saw it in about repo

**Comment by karzuo** [comment]
LGTM

**Review by karzuo — APPROVED** [review]
LGTM

**Review by ariefrahmansyah — APPROVED** [review]

**Pull request merged**